File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int result=0;
        int m = accounts.length;
        int n = accounts[0].length;
        for(int i=0;i<m;i++){
            int res =0;
            for(int j=0;j<n;j++){
                res+=accounts[i][j];
            }
            result = Math.max(res,result);
        }
        
        return result;
    }
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let ans = 0;
    for(let i=0;i<accounts.length;i++){
        let sum = 0;
        for(let j=0;j<accounts[0].length;j++){
            sum+=accounts[i][j];
        }
        ans = Math.max(sum,ans);
    }
    return ans;
};